pkg/pgp/key.go

```diff
@@ -107,6 +107,12 @@ func (p *Key) IsExpired(clockSkew time.Duration) bool {
 	now := time.Now()
 
 	i := p.key.GetEntity().PrimaryIdentity()
+	keyLifetimeSecs := i.SelfSignature.KeyLifetimeSecs
```

Comment:
Since this is an exported function I propose we forbid any negative clockSkew with panic guard.

Reply:
make sense, fixed